includes/rest-api/class-wp-rest-feature-controller.php

```diff
@@ -509,12 +509,6 @@ private function register_feature_routes( $feature ) {
 								),
 							),
 						),
-						'context' => array(
```

Comment:
There were some issues, particularly with GET requests and nesting this as a proper object. 

I decided to simplify things here and now the context can either be passed as simple query string arguments, or directly in the POST body.